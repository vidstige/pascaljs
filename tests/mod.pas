program mod;
begin
    WriteLn('44 mod 10 = ', 44 mod 10);
end.