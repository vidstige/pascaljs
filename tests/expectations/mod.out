44 mod 10 = 4
